docs/source/tutorial01.rst

```diff
@@ -1,4 +1,4 @@
-Tutorial 1: Using pyMOR’s discretization toolkit
+Tutorial: Using pyMOR’s discretization toolkit
```

Comment:
This is already part of #1021.

Reply:
That's right, thank you. I will remove it from this PR.